backend/config/curation-api.yml

```diff
@@ -46,20 +50,24 @@ paths:
 
   /v1/collections:
     get:
-      summary: Fetch Collections metadata.
-      description: >-
-        * When the visibility parameter is unspecified or set to `PUBLIC`, a list of all public collections is returned.
-        The Authorization header is not required. If a Collection in the list has been deleted, then it is annotated
-        with tombstone set to True.
+      summary: List Collections' metadata.
+      description: |
+        For each Collection in the list, the datasets for that Collection include a subset of dataset metadata as a 
```

Comment:
@brianraymor better?

Reply:
Better - but there's a [pending question](https://czi-sci.slack.com/archives/C02UGUV5R3J/p1662573600771709?thread_ts=1662572918.431129&cid=C02UGUV5R3J) about whether `tombstones` should have been disabled in the implementation and removed from the API.